src/eredis_client.erl

```diff
@@ -336,7 +338,7 @@ reply(Value, Queue) ->
             queue:in_r({N - 1, From, [Value | Replies]}, NewQueue);
         {empty, Queue} ->
             %% Oops
-            error_logger:info_msg("eredis: Nothing in queue, but got value from parser~n"),
+            ?LOG_NOTICE("eredis: Nothing in queue, but got value from parser~n"),
```

Comment:
```suggestion
            ?LOG_NOTICE("Nothing in queue, but got value from parser"),
```
In this specific case `~n` will be printed to log since no arguments are given.